src/certificate.rs

```diff
@@ -631,12 +637,7 @@ mod read_pki {
     /// }
     /// ```
     pub(super) fn ec_parameters(parameters: &DerObject<'_>) -> Result<AlgorithmId, Error> {
-        let curve_oid = match parameters.as_context_specific() {
-            Ok((_, Some(named_curve))) => {
-                named_curve.as_oid_val().map_err(|_| Error::InvalidObject)
-            }
-            _ => Err(Error::InvalidObject),
-        }?;
+        let curve_oid = parameters.as_oid_val().map_err(|_| Error::InvalidObject)?;
```

Comment:
@str4d does this seem OK to you? The `as_context_specific` method was removed in the latest `der_parser` release